src/Tests/Microsoft.NET.ToolPack.Tests/GivenThatWeWantToPackAToolProject.cs

```diff
@@ -37,12 +37,16 @@ private string SetupNuGetPackage(bool multiTarget, [CallerMemberName] string cal
                     if (multiTarget)
                     {
                         propertyGroup.Element(ns + "TargetFramework").Remove();
-                        propertyGroup.Add(new XElement(ns + "TargetFrameworks", "netcoreapp1.1;netcoreapp2.0"));
+                        propertyGroup.Add(new XElement(ns + "TargetFrameworks", "netcoreapp2.1"));
                     }
-                })
-                .Restore(Log);
+                });
+
+            List<string> sources = new List<string>() { NuGetConfigWriter.DotnetCoreMyGetFeed };
```

Comment:
I think this workaround will be unnecessary once we merge https://github.com/dotnet/sdk/pull/2157

Reply:
workaround removed